src/pages/layer-2.js

```diff
@@ -0,0 +1,1170 @@
+        alt: "Zapper",
+      },
+      {
+        title: "Zerion",
+        description:
+          "Build and manage your entire DeFi portfolio from one place. Discover the world of decentralized finance today.",
+        link: "https://zerion.io",
+        image: getImage(data.zerion),
+        alt: "Zerion",
+      },
+      {
+        title: "DeBank",
+        description:
+          "Keep up with all the important happenings in the web3 world",
+        link: "https://debank.com",
+        image: getImage(data.debank),
+        alt: "DeBank",
+      },
+    ],
+  }
+
+  const layer2DataCombined = [...layer2Data.optimistic, ...layer2Data.zk]
+
+  return (
+    <Page>
+      <PageMetadata
+        title={"Layer 2"}
+        description={"Introduction page to layer 2"}
+      />
+
+      <HeroBackground>
+        <HeroContainer>
+          <Hero content={heroContent} isReverse />
+        </HeroContainer>
+
+        <PaddedContent>
+          <StatsContainer>
+            <StatBox>
+              {/* TODO: remove hardcoded StatPrimary once lambda functions are working*/}
+              <StatPrimary>$7.27B</StatPrimary>
+              {/* <StatPrimary>{tvl}</StatPrimary>  */}
+              <StatDescription>TVL locked in layer 2 (USD)</StatDescription>
+            </StatBox>
+            <StatDivider />
+            <StatBox>
+              <StatPrimary>{averageFee}</StatPrimary>
+              <StatDescription>
+                Average layer 2 ETH transfer fee (USD)
+              </StatDescription>
+            </StatBox>
+            <StatDivider />
+            <StatBox>
+              {/* TODO: remove hardcoded StatPrimary once lambda functions are working*/}
+              <StatPrimary>33.01%</StatPrimary>
+              {/* <StatPrimary>{percentChangeL2}</StatPrimary> */}
+              <StatDescription>Layer 2 TVL change (30 days)</StatDescription>
+            </StatBox>
+          </StatsContainer>
+        </PaddedContent>
+      </HeroBackground>
+
+      <PaddedContent id="what-is-layer-2">
+        <TwoColumnContent>
+          <Flex50>
+            <h2>What is layer 2?</h2>
+            <p>
+              Layer 2 (L2) is a collective term to describe a specific set of
+              Ethereum scaling solutions.{" "}
+              <b>
+                A layer 2 is separate blockchain that extends Ethereum and
+                inherits the security guarantees of Ethereum
+              </b>
+              .
+            </p>
+            <p>
+              Now let’s dig into it a bit more, and to do this we need to
+              explain layer 1 (L1).
+            </p>
+          </Flex50>
+          <Flex50>
+            <GatsbyImage
+              image={getImage(data.whatIsEthereum)}
+              style={{ maxHeight: "400px" }}
+              objectFit="contain"
+            />
+          </Flex50>
+        </TwoColumnContent>
+      </PaddedContent>
+      <LightGrayContent>
+        <h2>Layer 1 function</h2>
```

Comment:
```suggestion
        <h2>What is Layer 1?</h2>
```
I still like `What is Layer 1?` or `Layer 1` here - `Layer 1 function` seems strange wording to me.